### ISSUE TITLE
Do not save in the db null option values

### ISSUE THREAD

**Issue by danyj** [ThemeFuse/Unyson#318]
I am trying to add just and empty title/separator/div in between the options sets to add some kind of hints, links to doc about the option sets, descriptions etc. 

I know I can make a custom option for it but this option should not be saved at all since it just displays info. What would be best approach ? 
let
    
    _get_value_from_input

return false, 

use only 
   
    _render

or only 

    _get_defaults

Basically what I am trying to figure out is which one of default option method should I leave out and this way not force it to be saved in DB.

**Comment by moldcraft** [comment]
We use [this function](https://github.com/ThemeFuse/Unyson/blob/c0b811bb14acbbe3a9c69874795d1628c9d4fd7e/framework/helpers/general.php#L723-L724) to extract option values from `$_POST`. As you can see, it will save `'option_id' => null` in the database if no value provided/found. So there is no way to prevent option value save in the database.

**Comment by danyj** [comment]
OK so there is no way to just add arbitrary text without saving the data correct?
This is what I am after. http://prntscr.com/6czzrv but I dont see a reason for these to be saved since they have no useful values.

**Comment by moldcraft** [comment]
> OK so there is no way to just add arbitrary text without saving the data correct?

Unfortunately, no.

----

Maybe I should change that function do not save `null` values?

```php
foreach (fw_extract_only_options($options) as $id => $option) {
	$values[$id] = fw()->backend->option_type($option['type'])->get_value_from_input(
		$option,
		isset($input_array[$id]) ? $input_array[$id] : null
	);

	if (is_null($values[$id])) {
		// do not save null values
		unset($values[$id]);
	}
}
```

There should be no side effects.

**Comment by danyj** [comment]
Hm , I'm thinking in what case do we leave it empty. I dont want to break anyone's options, but as you see makes no sense to save these. Maybe an attribute that will tell the method not to save it?

Well , we do use null values or should I say empty ones but looking at specs it should not hurt.
http://prntscr.com/6d08tx

**Comment by moldcraft** [comment]
Anyway [`null` values are treated like not existing values](https://github.com/ThemeFuse/Unyson/blob/e925c16f5929b46568429d6a6da8201b5b35bb81/framework/helpers/database.php#L23) so I will make the changes in function mentioned [above](https://github.com/ThemeFuse/Unyson/issues/318#issuecomment-77377794).
